### ISSUE TITLE
Retrieve/Stow Command

### ISSUE THREAD

**Issue by GoldenEagle75** [ThePix/QuestJS#87]
Hello Pixie,

Thank you (and the other contributors) so much for QuestJS! I never had a windows computer so previously when I used Quest 5 I had to use the slow online version which would often freeze. Now, with QuestJS and its code editable in basically any text editor, the problem is solved.

It appears that there is a small bug connected to the retrieve menu button with the RPG library. Whenever clicking the button, errors are generated. If you type "retrieve menu" many many errors are generated.
I added a line to settings.js (settings.storage=true) but this did not wholly fix it. The dialog box that opens up after typing "retrieve" is unescapable as far as I have seen and the "Confirm" button does not work. The "retrieve menu" command also causes many errors.

Thanks,
GoldenEagle75

**Comment by ThePix** [comment]
Hmm, looks like that is something I started to implement and then gave up on. Not sure why. I have it working, but there is something else I was working on, and I want to resolve that (at least to a degree), before I commit. Give me 2 days and it will be done. I will also document it - that will happen faster!

**Comment by ThePix** [comment]
I have uploaded to GitHub a new version, but not a new release. If you are not sure how to update from there, go to the "Code" tab, click on Code, and click "Download ZIP". Move your game files to a safe location (and back up!), then unzip the downloaded files. Then you can move your game files back.

I will leave this open for a week or so in case you find any issues.

**Comment by ThePix** [comment]
That is bizarre, because that is what it was doing for me BEFORE I updated it.

Can you play the example game in the "game-rpg" folder? This is what I tested with. If you append `?rpg` to the URL you should see it. How does that look?

Also, I am on Win10 in Chrome. I guess it is possible there is a browser issue here. Are you on Safari? It should not matter, but...

**Comment by GoldenEagle75** [comment]
Oh, sorry about that, it appears that the file I downloaded for this was the not the updated one but rather QuestJS-1-3.zip.
I tested with the RPG library from the RPG folder and the dialog box appears to work perfectly but the text processor says that it cannot find any such directive as "list". 
I found a list tutorial under the text processor section. Is this something you have added in the wiki but not committed to Github yet?
